mantis-runtime/src/main/java/io/mantisrx/runtime/sink/ServerSentEventRequestHandler.java

```diff
@@ -129,7 +129,7 @@ public Observable<Void> handle(HttpServerRequest<ByteBuf> request,
 
         // decouple the observable on a separate thread and add backpressure handling
         String decoupleSSE = "false";//ServiceRegistry.INSTANCE.getPropertiesService().getStringValue("sse.decouple", "false");
-        //Note: Below condition would be always false during if condition.
+        //Todo Note: Below condition would be always false during if condition.
```

Comment:
I think this is the syntax for a TODO comment. https://www.jetbrains.com/help/idea/using-todo.html

Reply:
Updated.